src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/AddArtworksModalContent.tsx

```diff
@@ -89,34 +117,83 @@ export const AddArtworksModalContent: FC<AddArtworksModalContentProps> = ({
         onItemClick={handleItemClick}
         selectedIds={selectedArtworkIds}
       />
+
+      {relay.hasMore() && <InfiniteScrollSentinel onNext={handleLoadMore} />}
+
+      {isLoadingNextPage && <ContentPlaceholder gridOnly={true} />}
     </>
   )
 }
 
-const AddArtworksModalContentRefetchContainer = createRefetchContainer(
+interface InfiniteScrollSentinelProps {
+  onNext(): void
+}
+
+const InfiniteScrollSentinel: FC<InfiniteScrollSentinelProps> = ({
+  onNext,
+}) => {
+  const { ref } = useIntersectionObserver({
+    onIntersection: onNext,
+    once: false,
+  })
+
+  return <Box ref={ref as any} height={0} />
```

Comment:
**question**: is this problem relevant [here](https://github.com/artsy/force/blob/33415594616757bfbbe5a6741874087bd30997b3/src/Components/Notifications/NotificationsListScrollSentinel.tsx#L16)?

Reply:
It works for me for some reason 🤷